tests/ember_debug/object-inspector-test.js

```diff
@@ -659,15 +660,28 @@ module('Ember Debug - Object Inspector', function (hooks) {
       3,
       'Correctly merges properties'
     );
-    assert.equal(message.details[3].properties[0].name, 'toString');
-    assert.equal(message.details[3].properties[1].name, 'hasChildren');
+
+    const toString = A(message.details[3].properties).findBy(
+      'name',
+      'toString'
+    );
+    const hasChildren = A(message.details[3].properties).findBy(
+      'name',
+      'hasChildren'
+    );
+    const expensiveProperty = A(message.details[3].properties).findBy(
+      'name',
+      'expensiveProperty'
+    );
```

Comment:
I think it would be easier to use `.find` directly.

```suggestion
    const toString = message.details[3].properties.find((p) => p.name === 'toString');
    const hasChildren = message.details[3].properties.find((p) => p.name === 'hasChildren');
    const expensiveProperty = message.details[3].properties.find((p) => p.name === 'expensiveProperty');
```